cli/server/spreadsheetRoutes.ts

```diff
@@ -789,53 +806,122 @@ function exportAsCSV(controls: any[], metadata: any, res: express.Response) {
 		Object.keys(control).forEach((key) => allFields.add(key));
```

Comment:
Commenting here because I'm not sure where it's most applicable. I think if the mappings is empty we should fallback to the original column/control field. So if I picked `test results` as my mapping column, but a given control didn't have any mappings, it should be populated with the original `test results`.